src/api/__tests__/bolt.spec.ts

```diff
@@ -0,0 +1,96 @@
+// import { PanelController, WindowController } from '../controllers';
```

Comment:
A whole file that is commented? *tut* *tut*

Reply:
it is not implemented yet, and will not be in the scope of this PR, however it was causing issues with the react compiler that were preventing me from building/running the project. Commented out until implementation is completed.